doubleml/irm/tests/test_ssm_tune.py

```diff
@@ -115,41 +115,73 @@ def dml_ssm_fixture(
     np.random.seed(42)
     smpls = all_smpls[0]
     if tune_on_folds:
-        g0_best_params, g1_best_params, pi_best_params, m_best_params = tune_nuisance_ssm(
-            y,
-            x,
-            d,
-            z,
-            s,
-            clone(learner_g),
-            clone(learner_m),
-            clone(learner_m),
-            smpls,
-            score,
```

Comment:
I removed the argument 'score' because there are 2 different function for each score